backend/typescript/services/implementations/camperService.ts

```diff
@@ -637,6 +639,53 @@ class CamperService implements ICamperService {
     }
   }
 
+  /* eslint-disable consistent-return */
+  async inviteWaitlistedCamper(waitlistedCamperId: string): Promise<any> {
```

Comment:
can we return errors if camp, campSession or camperToUpdate aren't found? the request is still successful if one of those are null